fft-test.c

```diff
@@ -0,0 +1,52 @@
+#define FFT_WRITE_LANE  0x2000
```

Comment:
This file should be moved into Chipyards `tests/` directory. https://github.com/ucb-bar/chipyard/tree/master/tests